include/sys/device_mmio.h

```diff
@@ -15,7 +15,6 @@
 #ifndef ZEPHYR_INCLUDE_SYS_DEVICE_MMIO_H
 #define ZEPHYR_INCLUDE_SYS_DEVICE_MMIO_H
 
-#include <toolchain/common.h>
```

Comment:
The file seems to make use of `ARG_UNUSED`, `<toolchain.h>` is probably the right header then?